mephisto/abstractions/providers/mock/wrap_crowd_source.js

```diff
@@ -45,6 +45,12 @@ function handleSubmitToProvider(task_data) {
   return true;
 }
 
+window.HIDE_SUBMIT_BUTTON = (isHidden) => {
+  const submitButton = document.getElementById("html-task-submit-button");
+  if (isHidden) submitButton.style.display = "none";
+  else submitButton.style.display = "block";
```

Comment:
Hmm will this actually work? If we modify the element like this outside of React, then if React ever re-renders the above component, there's no guarantee that this style stays, and will actually most likely be wiped out. Sounds like this could lead to a really confusing bug down the road.